src/ia-menu-slider.js

```diff
@@ -121,18 +125,22 @@ export class IAMenuSlider extends LitElement {
   /** @inheritdoc */
   render() {
     return html`
+      <div class="main">
       <div class="menu ${this.sliderDetailsClass}">
         ${this.closeButton}
         <ul class="menu-list">
           ${this.menuItems}
         </ul>
         <div class="content ${this.selectedMenuClass}" @menuTypeSelected=${this.setSelectedMenu}>
           ${this.renderMenuHeader}
-          <section class="selected-menu">
-            ${this.selectedMenuComponent}
+          <section>
```

Comment:
Do we really need empty section HTML tag, can't we just use the below class or div tag?